independent-publisher-connectors/DPIRD Weather/Readme.md

```diff
@@ -0,0 +1,139 @@
+

```

Comment:
Readme doesnt have all required sections. Please follow this [link ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)for template.